### ISSUE TITLE
Compil pjsip for Android failed

### ISSUE THREAD

**Issue by TienHai** [pjsip/pjproject#2925]
### Describe the bug

Hi, 

I try to compil pjsip for android, but failed.

### Steps to reproduce

Source code from github, branch master

export ANDROID_NDK_ROOT=/Users/tien/Library/Android/sdk/ndk/23.1.7779620

vi pjlib/include/pj/config_site.h
=> add : 
#define PJ_CONFIG_ANDROID 1
#define PJMEDIA_HAS_OPUS_CODEC 1
#include <pj/config_site_sample.h>

APP_PLATFORM=android-30 TARGET_ABI=arm64-v8a ./configure-android --with-opus=/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib

make dep && make clean && make

### PJSIP version

master

### Context

mac, branch master, opus 1.3.1

### Log, call stack, etc

```shell
APP_PLATFORM=android-30 TARGET_ABI=arm64-v8a ./configure-android --with-opus=/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib
configure-android: calling ./configure with env vars:
 NDK_TOOLCHAIN = 
 CC = /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang
 CXX = /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++
 CFLAGS =  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security  
 CXXFLAGS =  -fno-exceptions -fno-rtti -fexceptions -frtti  
 LDFLAGS =  /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/lib64/clang/12.0.8/lib/linux/aarch64/libunwind.a -latomic -target aarch64-none-linux-android30 -no-canonical-prefixes -Wl,--build-id=sha1 -nostdlib++ -Wl,--no-undefined -Wl,--fatal-warnings -lc -lm  
 LIBS =  /Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/libs/arm64-v8a/libc++_shared.so
 AR = /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar
 RANLIB = /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar s
 TARGET_HOST = aarch64-linux-android
 TARGET_ABI = arm64-v8a
checking build system type... x86_64-apple-darwin21.1.0
checking host system type... aarch64-unknown-linux-android
checking target system type... aarch64-unknown-linux-android
checking for aarch64-linux-android-gcc... /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang accepts -g... yes
checking for /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang option to accept ISO C89... none needed
checking whether we are using the GNU C++ compiler... yes
checking whether /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++ accepts -g... yes
checking for aarch64-linux-android-ranlib... /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar s
checking for aarch64-linux-android-ar... /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar
checking for pthread_create in -lpthread... no
checking for puts in -lwsock32... no
checking for puts in -lws2_32... no
checking for puts in -lole32... no
checking for puts in -lwinmm... no
checking for puts in -lsocket... no
checking for puts in -lrt... no
checking for sin in -lm... yes
checking for uuid_generate in -luuid... no
checking for uuid_generate in -luuid... (cached) no
checking for library containing gethostbyname... none required
Setting PJ_M_NAME to aarch64
checking memory alignment... 8 bytes
checking how to run the C preprocessor... /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... no
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking whether byte ordering is bigendian... no
Checking if floating point is disabled... no
checking arpa/inet.h usability... yes
checking arpa/inet.h presence... no
aconfigure: WARNING: arpa/inet.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: arpa/inet.h: proceeding with the compiler's result
checking for arpa/inet.h... yes
checking assert.h usability... yes
checking assert.h presence... no
aconfigure: WARNING: assert.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: assert.h: proceeding with the compiler's result
checking for assert.h... yes
checking ctype.h usability... yes
checking ctype.h presence... no
aconfigure: WARNING: ctype.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: ctype.h: proceeding with the compiler's result
checking for ctype.h... yes
checking errno.h usability... yes
checking errno.h presence... no
aconfigure: WARNING: errno.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: errno.h: proceeding with the compiler's result
checking for errno.h... yes
checking fcntl.h usability... yes
checking fcntl.h presence... no
aconfigure: WARNING: fcntl.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: fcntl.h: proceeding with the compiler's result
checking for fcntl.h... yes
checking linux/socket.h usability... yes
checking linux/socket.h presence... no
aconfigure: WARNING: linux/socket.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: linux/socket.h: proceeding with the compiler's result
checking for linux/socket.h... yes
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking malloc.h usability... yes
checking malloc.h presence... no
aconfigure: WARNING: malloc.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: malloc.h: proceeding with the compiler's result
checking for malloc.h... yes
checking netdb.h usability... yes
checking netdb.h presence... no
aconfigure: WARNING: netdb.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: netdb.h: proceeding with the compiler's result
checking for netdb.h... yes
checking netinet/in_systm.h usability... yes
checking netinet/in_systm.h presence... no
aconfigure: WARNING: netinet/in_systm.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: netinet/in_systm.h: proceeding with the compiler's result
checking for netinet/in_systm.h... yes
checking netinet/in.h usability... yes
checking netinet/in.h presence... no
aconfigure: WARNING: netinet/in.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: netinet/in.h: proceeding with the compiler's result
checking for netinet/in.h... yes
checking for netinet/ip.h... yes
checking netinet/tcp.h usability... yes
checking netinet/tcp.h presence... no
aconfigure: WARNING: netinet/tcp.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: netinet/tcp.h: proceeding with the compiler's result
checking for netinet/tcp.h... yes
checking ifaddrs.h usability... yes
checking ifaddrs.h presence... no
aconfigure: WARNING: ifaddrs.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: ifaddrs.h: proceeding with the compiler's result
checking for ifaddrs.h... yes
checking for getifaddrs... yes
checking semaphore.h usability... yes
checking semaphore.h presence... no
aconfigure: WARNING: semaphore.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: semaphore.h: proceeding with the compiler's result
checking for semaphore.h... yes
checking setjmp.h usability... yes
checking setjmp.h presence... no
aconfigure: WARNING: setjmp.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: setjmp.h: proceeding with the compiler's result
checking for setjmp.h... yes
checking stdarg.h usability... yes
checking stdarg.h presence... yes
checking for stdarg.h... yes
checking stddef.h usability... yes
checking stddef.h presence... yes
checking for stddef.h... yes
checking stdio.h usability... yes
checking stdio.h presence... no
aconfigure: WARNING: stdio.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: stdio.h: proceeding with the compiler's result
checking for stdio.h... yes
checking for stdint.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking sys/ioctl.h usability... yes
checking sys/ioctl.h presence... no
aconfigure: WARNING: sys/ioctl.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: sys/ioctl.h: proceeding with the compiler's result
checking for sys/ioctl.h... yes
checking sys/select.h usability... yes
checking sys/select.h presence... no
aconfigure: WARNING: sys/select.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: sys/select.h: proceeding with the compiler's result
checking for sys/select.h... yes
checking sys/socket.h usability... yes
checking sys/socket.h presence... no
aconfigure: WARNING: sys/socket.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: sys/socket.h: proceeding with the compiler's result
checking for sys/socket.h... yes
checking sys/time.h usability... yes
checking sys/time.h presence... no
aconfigure: WARNING: sys/time.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: sys/time.h: proceeding with the compiler's result
checking for sys/time.h... yes
checking sys/timeb.h usability... no
checking sys/timeb.h presence... no
checking for sys/timeb.h... no
checking for sys/types.h... (cached) yes
checking sys/filio.h usability... no
checking sys/filio.h presence... no
checking for sys/filio.h... no
checking sys/sockio.h usability... no
checking sys/sockio.h presence... no
checking for sys/sockio.h... no
checking sys/utsname.h usability... yes
checking sys/utsname.h presence... no
aconfigure: WARNING: sys/utsname.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: sys/utsname.h: proceeding with the compiler's result
checking for sys/utsname.h... yes
checking time.h usability... yes
checking time.h presence... no
aconfigure: WARNING: time.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: time.h: proceeding with the compiler's result
checking for time.h... yes
checking for unistd.h... (cached) yes
checking winsock.h usability... no
checking winsock.h presence... no
checking for winsock.h... no
checking winsock2.h usability... no
checking winsock2.h presence... no
checking for winsock2.h... no
checking for mswsock.h... no
checking ws2tcpip.h usability... no
checking ws2tcpip.h presence... no
checking for ws2tcpip.h... no
checking uuid/uuid.h usability... no
checking uuid/uuid.h presence... no
checking for uuid/uuid.h... no
checking for net/if.h... yes
checking linux/android_alarm.h usability... no
checking linux/android_alarm.h presence... no
checking for linux/android_alarm.h... no
checking for localtime_r... yes
Setting PJ_OS_NAME to aarch64-unknown-linux-android
Setting PJ_HAS_ERRNO_VAR to 1
Setting PJ_HAS_HIGH_RES_TIMER to 1
Setting PJ_HAS_MALLOC to 1
Setting PJ_NATIVE_STRING_IS_UNICODE to 0
Setting PJ_ATOMIC_VALUE_TYPE to long
checking if inet_aton() is available... yes
checking if inet_pton() is available... yes
checking if inet_ntop() is available... yes
checking if getaddrinfo() is available... yes
checking if sockaddr_in has sin_len member... no
checking if socklen_t is available... yes
checking if IPV6_V6ONLY is available... yes
checking if SO_ERROR is available... yes
checking if pthread_rwlock_t is available... yes
checking if pthread_mutexattr_settype() is available... no
checking if pthread_mutexattr_t has recursive member... no
checking ioqueue backend... select()
Building shared libraries... no
Building pjsua2 library and application... yes
checking sys/soundcard.h usability... no
checking sys/soundcard.h presence... no
checking for sys/soundcard.h... no
checking linux/soundcard.h usability... yes
checking linux/soundcard.h presence... no
aconfigure: WARNING: linux/soundcard.h: accepted by the compiler, rejected by the preprocessor!
aconfigure: WARNING: linux/soundcard.h: proceeding with the compiler's result
checking for linux/soundcard.h... yes
checking machine/soundcard.h usability... no
checking machine/soundcard.h presence... no
checking for machine/soundcard.h... no
Checking sound device backend... OpenSL ES
Checking if OpenGL ES 2 is available... no
Checking if small filter is disabled... no
Checking if large filter is disabled... no
Checking if Speex AEC is disabled...no
Checking if G.711 codec is disabled...no
Checking if L16 codec is disabled...no
Checking if GSM codec is disabled...no
Checking if G.722 codec is disabled...no
Checking if G.722.1 codec is disabled...no
Checking if Speex codec is disabled...no
Checking if iLBC codec is disabled...no
Checking if libsamplerate is enabled...no
Building libresample as shared library... no
Checking if Speex resample is enabled... no
Checking if SDL is disabled... yes
Checking if ffmpeg is disabled... yes
checking for v4l2_open in -lv4l2... no
Checking if OpenH264 is disabled... yes
Checking if VPX is disabled... yes
Skipping Intel IPP settings (not wanted)
checking for AMediaCodec_createDecoderByType in -lmediandk... yes
Checking if Android AMediaCodec library is available... yes
Checking if SSL support is disabled... yes
Checking if OpenCORE AMR support is disabled... yes
Checking if SILK support is disabled... yes
checking for OPUS installations..
Using OPUS prefix... /Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib
checking opus/opus.h usability... yes
checking opus/opus.h presence... yes
checking for opus/opus.h... yes
checking for opus_repacketizer_get_size in -lopus... yes
OPUS library found, OPUS support enabled
Checking if bcg729 is disabled... yes
Checking if libyuv is disabled...no
Checking if libwebrtc is disabled...no
Checking if libwebrtc-aec3 is enabled...no
checking if select() needs correct nfds... no (default)
** Decided that select() doesn't need correct nfds (please check)
checking if pj_thread_create() should enforce stack size... no (default)
checking if pj_thread_create() should allocate stack... no (default)
** Setting non-blocking recv() retval to EAGAIN (please check)
** Setting non-blocking connect() retval to EINPROGRESS (please check)
aconfigure: creating ./config.status
config.status: creating build.mak
config.status: creating build/os-auto.mak
config.status: creating build/cc-auto.mak
config.status: creating pjlib/build/os-auto.mak
config.status: creating pjlib-util/build/os-auto.mak
config.status: creating pjmedia/build/os-auto.mak
config.status: creating pjsip/build/os-auto.mak
config.status: creating third_party/build/os-auto.mak
config.status: creating pjlib/include/pj/compat/os_auto.h
config.status: creating pjlib/include/pj/compat/m_auto.h
config.status: creating pjmedia/include/pjmedia/config_auto.h
config.status: creating pjmedia/include/pjmedia-codec/config_auto.h
config.status: creating pjsip/include/pjsip/sip_autoconf.h


Configurations for current target have been written to 'build.mak', and 'os-auto.mak' in various build directories, and pjlib/include/pj/compat/os_auto.h.

Further customizations can be put in:
  - 'user.mak'
  - 'pjlib/include/pj/config_site.h'

The next step now is to run 'make dep' and 'make'.


make dep && make clean && make

....
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++ -c  -DPJ_AUTOCONF=1  -fno-exceptions -fno-rtti -fexceptions -frtti      -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include                 \
		-o output/pjsua2-lib-aarch64-unknown-linux-android/siptypes.o \
		../src/pjsua2/siptypes.cpp
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++ -c  -DPJ_AUTOCONF=1  -fno-exceptions -fno-rtti -fexceptions -frtti      -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include                 \
		-o output/pjsua2-lib-aarch64-unknown-linux-android/call.o \
		../src/pjsua2/call.cpp
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++ -c  -DPJ_AUTOCONF=1  -fno-exceptions -fno-rtti -fexceptions -frtti      -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include                 \
		-o output/pjsua2-lib-aarch64-unknown-linux-android/presence.o \
		../src/pjsua2/presence.cpp
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/clang++ -c  -DPJ_AUTOCONF=1  -fno-exceptions -fno-rtti -fexceptions -frtti      -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include -Wall -DPJ_AUTOCONF=1  -target aarch64-none-linux-android30 -fdata-sections -ffunction-sections -funwind-tables -no-canonical-prefixes --sysroot /Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/sysroot -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -D_FORTIFY_SOURCE=2 -fpic -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/include -I/Users/tien/Library/Android/sdk/ndk/23.1.7779620/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/include -Ijni -DANDROID -nostdinc++ -Wformat -Werror=format-security   -DPJ_IS_BIG_ENDIAN=0 -DPJ_IS_LITTLE_ENDIAN=1 -I/Users/tien/Devel/pjsip/opus-1.3.1/opus-dev-lib/include    -I../include -I../../pjlib/include -I../../pjlib-util/include -I../../pjnath/include -I../../pjmedia/include                 \
		-o output/pjsua2-lib-aarch64-unknown-linux-android/media.o \
		../src/pjsua2/media.cpp
In file included from ../src/pjsua2/media.cpp:20:
../include/pjsua2/media.hpp:2013:22: warning: private field 'winId' is not used [-Wunused-private-field]
    pjsua_vid_win_id winId;
                     ^
1 warning generated.
if test ! -d ../lib; then mkdir -p ../lib; fi
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar rv ../lib/libpjsua2-aarch64-unknown-linux-android.a  output/pjsua2-lib-aarch64-unknown-linux-android/account.o  output/pjsua2-lib-aarch64-unknown-linux-android/endpoint.o  output/pjsua2-lib-aarch64-unknown-linux-android/json.o  output/pjsua2-lib-aarch64-unknown-linux-android/persistent.o  output/pjsua2-lib-aarch64-unknown-linux-android/types.o  output/pjsua2-lib-aarch64-unknown-linux-android/siptypes.o  output/pjsua2-lib-aarch64-unknown-linux-android/call.o  output/pjsua2-lib-aarch64-unknown-linux-android/presence.o  output/pjsua2-lib-aarch64-unknown-linux-android/media.o
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar: warning: creating ../lib/libpjsua2-aarch64-unknown-linux-android.a
/Users/tien/Library/Android/sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/darwin-x86_64/bin/llvm-ar s ../lib/libpjsua2-aarch64-unknown-linux-android.a
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f Samples.mak
make[2]: Nothing to be done for `all'.
```

**Comment by goffioul** [comment]
I think you need to use this in your config_site.h:
```
#define PJ_CONFIG_ANDROID 1
#define PJMEDIA_HAS_VIDEO 1
#include <pj/config_site_sample.h>
```
`PJMEDIA_HAS_OPUS_CODEC` will get defined if opus detection succeeds.

**Comment by sauwming** [comment]
Should have been fixed by #2974. Please reopen if the issue persists.
